tests/storage/blocksd_test.py

```diff
@@ -862,6 +863,43 @@ def test_volume_metadata(tmp_storage, tmp_repo, fake_access, fake_rescan,
     assert data == md.storage_format(domain_version, CAP=md.capacity)
 
 
+@requires_root
+@pytest.mark.root
+@pytest.mark.xfail(reason='Vdsm over-allocates one chunk', strict=False)
+@pytest.mark.parametrize("capacity", [
+    # capacity
+    1 * MiB,
+    512 * MiB,
+    sc.VG_EXTENT_SIZE,
+    CHUNK_SIZE_MB * MiB - sc.VG_EXTENT_SIZE,
+    (CHUNK_SIZE_MB + 1) * MiB - sc.VG_EXTENT_SIZE,
+    (CHUNK_SIZE_MB - 1) * MiB,
+])
+def test_cow_volume_small_size(
```

Comment:
test_cow_small_volume?

A comment explaining what is small volume can help (e.g. capacity < chunk size)